### ISSUE TITLE
Can't return values from memoized functions

### ISSUE THREAD

**Issue by arfordweb** [stagas/memoize#1]
Is it intentional that you can't return values from the function you are memoizing?  Other libraries allow this.

Also, I recognize this module was written a long time ago.  But, it's towards the top of NPM's search results.  If this module is no longer maintained, it may be good to note this in the README.

**Comment by stagas** [comment]
@arfordweb Yes, it memoizes callbacks, not returns. It is maintained, it's just not actively developed. If you want to contribute some functionality it would be very welcome and accepted.
